src/helm/config/model_metadata.yaml

```diff
@@ -637,6 +637,40 @@ models:
     release_date: 2023-12-13
     tags: [VISION_LANGUAGE_MODEL_TAG, GOOGLE_GEMINI_MODEL_TAG, FULL_FUNCTIONALITY_VLM_TAG]
 
+  - name: google/gemma-2b
+    display_name: Gemma (2B)
+    description: Gemma is a family of lightweight, state-of-the-art open models from Google, built from the same research and technology used to create the Gemini models.
```

Comment:
I know we are taking this copy from the organization's website, but in this case, this is very Google-marketing-speak, which will look weird on the HELM website in a table with a list of description of all the models.  I'd suggest a more objective description.